cocos/video/video-player-impl-web.ts

```diff
@@ -165,6 +165,7 @@ export class VideoPlayerImplWeb extends VideoPlayerImpl {
     public seekTo (val: number): void {
         if (this.video) {
             this.video.currentTime = val;
+            this._cachedCurrentTime = this.video.currentTime;
```

Comment:
The reason is `_cachedCurrentTime ` in impl is not updated, but used in `syncCurrentTime` when play.
![image](https://github.com/cocos/cocos-engine/assets/18634782/73c63374-2498-4987-a928-bb4c3c9797df)
![image](https://github.com/cocos/cocos-engine/assets/18634782/a9138b95-051c-480c-9778-05f2a8bd6b51)